lib/deploy/plugin.js

```diff
@@ -150,10 +150,10 @@ module.exports = class AddonDocsDeployPlugin {
 
   _updateIndexContents(context, stagingDirectory, appRoot, deployVersion) {
     let indexPath = `${stagingDirectory}/${appRoot}/index.html`;
-    let rootURL = [this._getRootURL(), appRoot].filter(Boolean).join('/');
+    let rootURL = [this._getRootURL(), appRoot].filter(Boolean).join('/').replace(/^\/?([^/]+(?:\/[^/]+)*)\/?$/, '/$1/') || '/';
```

Comment:
Think you could add a comment explaining this regex? It's pretty complicated, having a hard time trying to parse it and understand what it's doing

Reply:
I will, actually I linked to the stack overflow post about it in the commit message.
https://github.com/ember-learn/ember-cli-addon-docs/pull/266/commits/381f17e9c8bf5433ff2bbac1905443291a7ce934

Reply:
ahh gotcha, missed that. I think the comment is good anyways, thanks for this!